mmv1/third_party/terraform/services/integrationconnectors/resource_integration_connectors_endpoint_attachment_test.go

```diff
@@ -74,6 +75,7 @@ resource "google_integration_connectors_endpoint_attachment" "sampleendpointatta
   labels = {
     bar = "foo"
   }
+  endpoint_global_access = false
```

Comment:
I would suggest changing this to `true`, to verify that a value other than false works, and to verify that the value can be updated in place. If it can't be updated in-place, then instead it should be configured in the yaml with `immutable: true`

Reply:
Done